jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/TestPlugin.java

```diff
@@ -42,6 +42,8 @@ protected void before() throws IOException, XmlPullParserException, Verification
     Verifier verifier = new Verifier(".", true);
     verifier.setAutoclean(false);
     verifier.addCliOption("-DskipTests");
+    verifier.addCliOption("-Dfmt.skip");
```

Comment:
Are these necessary?

Reply:
They let you run the plugin dependent tests from the IDE faster without having to invoke fmt/checkstyle.  fmt/checkstyle will still run in the end. But theres no reason for us to double run this. It make the installs run faster as well.